pytest_mergify/utils.py

```diff
@@ -65,6 +65,20 @@ def get_repository_name_from_env_url(env: str) -> typing.Optional[str]:
     return None
 
 
+class InvalidRepositoryNameError(Exception):
+    pass
+
+
+def split_full_repo_name(
+    full_repo_name: str,
+) -> typing.Tuple[typing.Optional[str], typing.Optional[str]]:
```

Comment:
I'm not sure to understand why the elements of the tuple are optional. I think it could be:
```suggestion
) -> typing.Tuple[str, str]:
```
Or, even:
```suggestion
) -> typing.Tuple[github_types.GitHubLogin, github_types.GitHubRepositoryName]:
```

WDYT?